homeassistant/components/netatmo/camera.py

```diff
@@ -101,6 +103,11 @@ async def get_entities():
             {vol.Optional(ATTR_PERSON): cv.string},
             "_service_set_person_away",
         )
+        platform.async_register_entity_service(
+            SERVICE_SET_CAMERA_LIGHT,
+            {vol.Required(ATTR_CAMERA_LIGHT_MODE): cv.string},
```

Comment:
Are only the listed three light modes available or can you define custom light modes? If the first is the case I would suggest to use vol.In() and define a constant list with the valid light modes.

Reply:
Thanks @springstan. Much better.